src/Calypso-SystemTools-Core/ClyClassDefinitionContext.class.st

```diff
@@ -5,3 +5,9 @@ Class {
 	#package : 'Calypso-SystemTools-Core',
 	#tag : 'Editors-Classes'
 }
+
+{ #category : 'testing' }
+ClyClassDefinitionContext >> isInstanceSide [
```

Comment:
I found this one strange. 
And I also wondered why we could not use metaLevelOf: even if I do not like the name.